MonoGame.Framework/Audio/SoundEffect.XAudio.cs

```diff
@@ -73,15 +73,6 @@ internal static void InitializeSoundEffect()
             {
                 if (Device == null)
                 {
-#if !WINRT && DEBUG
-                    try
-                    {
-                        //Fails if the XAudio2 SDK is not installed
-                        Device = new XAudio2(XAudio2Flags.DebugEngine, ProcessorSpecifier.DefaultProcessor);
-                        Device.StartEngine();
-                    }
-                    catch
-#endif
```

Comment:
Seems to have been removed in Windows 8 desktop: http://sharpdx.org/documentation/api/t-sharpdx-xaudio2-xaudio2flags